actions/lib/action.py

```diff
@@ -10,10 +10,29 @@ def __init__(self, config):
 
     def _get_client(self):
         url = self.config['url']
-        token = self.config['token']
         verify = self._get_verify()
 
+        auth_method = self.config.get("auth_method", "token")
+        token = self.config["token"]
+
+        # token is passed during client init to allow client to also
+        # get the token from VAULT_TOKEN env var or ~/.vault-token
         client = hvac.Client(url=url, token=token, verify=verify)
+
+        # token is handled during client init
+        # other auth methods will override it as needed
+        if auth_method == "token":
+            pass
+
```

Comment:
lots of extra space here makes it look like the `if` and `elif` aren't connected.